rpcs3/Emu/RSX/VK/VKHelpers.h

```diff
@@ -535,11 +535,11 @@ namespace vk
 			LOG_NOTICE(RSX, "Found vulkan-compatible GPU: '%s' running on driver %s", get_name(), get_driver_version());
 
 			if (get_driver_vendor() == driver_vendor::RADV &&
-				get_name().find("LLVM 8") != std::string::npos)
+				get_name().find("LLVM 8.0.0") != std::string::npos)
 			{
 				// Serious driver bug causing black screens
 				// See https://bugs.freedesktop.org/show_bug.cgi?id=110970
-				LOG_FATAL(RSX, "RADV drivers have a major driver bug with LLVM 8 resulting in no visual output. Upgrade to LLVM 9 version of mesa to avoid this issue.");
+				LOG_FATAL(RSX, "RADV drivers have a major driver bug with LLVM 8.0.0 resulting in no visual output. Upgrade to an LLVM 8.0.1 or 9.0.0 version of mesa to avoid this issue.");
```

Comment:
Minor nitpick in grammar for the log. 

```suggestion
				LOG_FATAL(RSX, "RADV drivers have a major driver bug with LLVM 8.0.0 resulting in no visual output. Upgrade to either LLVM version 8.0.1 or 9.0.0 of mesa to avoid this issue.");
```

Reply:
`Upgrade to llvm version 8.0.1 or greater` would convey the same message in fewer words. Or even just `Upgrade to any newer version of mesa` since only 8.0.0 is affected.

Reply:
Fixed